<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Midi::FileImportDialog</class>
 <widget class="QWidget" name="Midi::FileImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>223</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="midiFileGroupBox">
        <property name="title">
         <string>MIDI File</string>
        </property>
        <layout class="QHBoxLayout">
         <property name="margin">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="midiFileLayout" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="midiFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="midiFileOpenButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../mi_gui/mi_gui.qrc">
               <normaloff>:/mi_gui/images/fileopen.png</normaloff>:/mi_gui/images/fileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="importSettingsGroupBox">
        <property name="title">
         <string>Import Settings</string>
        </property>
        <layout class="QVBoxLayout">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="importTypeLayout" stretch="0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="importTypeComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <item>
              <property name="text">
               <string>Create New Score</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add to Current Score</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="importBarlinesLayout" stretch="0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="importBarlinesComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>96</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <item>
              <property name="text">
               <string>Import Barlines</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ignore Barlines</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../mi_gui/mi_gui.qrc"/>
 </resources>
 <connections/>
</ui>
